apps/ai-interviewer-agent/.env.example

```
@@ -0,0 +1,7 @@
+LIVEKIT_API_KEY="your_livekit_key_here"
+LIVEKIT_API_SECRET="your_livekit_secret_here"
+LIVEKIT_URL="wss://your_project.livekit.cloud"
+
+GROQ_API_KEY="your_grok_key_here"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)